### PR TITLE
feat(taro-lnd-link-details): info tab for taro-lnd link

### PR DESCRIPTION
**Pull request by amovfx**
### Description

Added basic taro-lnd peer link info.
Added test for taro-lnd link.
Updated LinkDetails tests for taro nodes.

### Steps to Test

1. Create a network with Taro
2. Click the link between the taro node and lnd node.

### CODE REVIEW

**Review by amovfx — COMMENTED** [review]
Reviewed. I didn't really have any ideas for the taro lnd link node, so I copied the Peer component.

File: src/i18n/locales/en-US.json [hunk 1]
```diff
@@ -147,6 +147,14 @@
   "cmps.designer.link.Peer.implementation": "Implementation",
   "cmps.designer.link.Peer.version": "Version",
   "cmps.designer.link.Peer.status": "Status",
+  "cmps.designer.link.lndbackend.title": "Taro Lnd Connection",
+  "cmps.designer.link.lndbackend.desc": "Taro nodes communicate over lnd to mint and transfer assets.",
+  "cmps.designer.link.lndbackend.taroTitle": "Taro Node",
+  "cmps.designer.link.lndbackend.lndTitle": "Lnd Node",
```
Comment (jamaljsr) [inline src/i18n/locales/en-US.json:153]: Rename "Lnd" to "LND". It should either be in all caps when referring to the name or all lower when referring to the CLI command.

**Review by jamaljsr — COMMENTED** [review]
Overall this works as expected. Just had a few comments on the code and labels.

Thanks for adding the tests to keep the coverage up. 👌

File: src/i18n/locales/en-US.json (hunk 1 above)
Comment (amovfx) [reply src/i18n/locales/en-US.json:153]: Done

File: src/i18n/locales/en-US.json [hunk 2]
```diff
@@ -147,6 +147,14 @@
   "cmps.designer.link.Peer.implementation": "Implementation",
   "cmps.designer.link.Peer.version": "Version",
   "cmps.designer.link.Peer.status": "Status",
+  "cmps.designer.link.lndbackend.title": "Taro Lnd Connection",
+  "cmps.designer.link.lndbackend.desc": "Taro nodes communicate over lnd to mint and transfer assets.",
```
[the remark this answers is not in the document]
Comment (amovfx) [reply src/i18n/locales/en-US.json:151]: Done

File: src/i18n/locales/en-US.json [hunk 3]
```diff
@@ -147,6 +147,14 @@
   "cmps.designer.link.Peer.implementation": "Implementation",
   "cmps.designer.link.Peer.version": "Version",
   "cmps.designer.link.Peer.status": "Status",
+  "cmps.designer.link.lndbackend.title": "Taro Lnd Connection",
```
[the remark this answers is not in the document]
Comment (amovfx) [reply src/i18n/locales/en-US.json:150]: Done

File: src/components/designer/link/TaroLnd.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import DetailsList, { DetailValues } from 'components/common/DetailsList';
+import SidebarCard from '../SidebarCard';
+
+interface Props {
+  from: TaroNode;
+  to: LndNode;
+}
+
+const TaroLnd: React.FC<Props> = ({ from, to }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.link.lndbackend');
```
[the remark this answers is not in the document]
Comment (amovfx) [reply src/components/designer/link/TaroLnd.tsx:14]: Done

File: src/components/designer/link/TaroLnd.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import { StatusBadge } from 'components/common';
+import DetailsList, { DetailValues } from 'components/common/DetailsList';
+import SidebarCard from '../SidebarCard';
+
+interface Props {
+  from: TaroNode;
+  to: LndNode;
+}
+
+const TaroLnd: React.FC<Props> = ({ from, to }) => {
```
[the remark this answers is not in the document]
Comment (amovfx) [reply src/components/designer/link/TaroLnd.tsx:13]: Done

File: src/components/designer/link/TaroBackend.tsx [hunk 6]
```diff
@@ -10,8 +10,8 @@ interface Props {
   to: LndNode;
 }
 
-const TaroLnd: React.FC<Props> = ({ from, to }) => {
-  const { l } = usePrefixedTranslation('cmps.designer.link.lndbackend');
+const TaroBackend: React.FC<Props> = ({ from, to }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.link.tarobackend');
```
Comment (jamaljsr) [inline src/components/designer/link/TaroBackend.tsx:14]: The key should match the name of the component, case-sensitive.

Comment (amovfx) [reply src/components/designer/link/TaroBackend.tsx:14]: Done

**Comment by amovfx** [comment]
Is there an easy way to move this over to draft? I decided to add the change TaroBackend ability to this pr. Or should that be a separate pr?

**Comment by jamaljsr** [comment]
There is a "Convert to draft" link under the Reviewers section of this page.

Definitely create a separate PR. These two features are unrelated.

**Comment by amovfx** [comment]
Okay, thanks for the guidance.

**Review by jamaljsr — APPROVED** [review]
tACK 👌

Great job on the tests.

**Pull request merged**